src/Castle.Windsor/MicroKernel/Lifestyle/Scoped/IScopeAccessor.cs

```diff
@@ -32,5 +32,9 @@ public interface IScopeAccessor : IDisposable
 		/// <param name = "context">Current creation context</param>
 		/// <exception cref = "T:System.InvalidOperationException"> Thrown when scope cache could not be accessed.</exception>
 		ILifetimeScope GetScope(CreationContext context);
+		ILifetimeScope GetScope(CreationContext context, IKernel kernel);
+		bool HasKernelScoping { get; }
```

Comment:
This is obviously a breaking change to `IScopeAccessor`, implemented by both Windsor facilities and users. It also is counter to the goal of scopes within a container.